lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1758,7 +1788,7 @@ def remove_ip_blocks(self, blocks):
             removed_blocks: (tuple(ip_network)): tuple of ipaddress.ip_netework
                 objects representing the removed IP blocks.
         """
-        removed_blocks = self._mobility_client.remove_ip_blocks(blocks)
+        removed_blocks = self._mobility_client.remove_ip_blocks(blocks, force=True)
```

Comment:
This is for test cleanup (this function is only used in  `MobilityUtil.cleanup`), so that all allocated IPs and IP blocks are removed at the end of each test no matter what (otherwise some stuff will be left in the redis)